packages/storybook-builder-parcel/preset.js

```diff
@@ -119,10 +119,14 @@ async function createParcel(options, isDev = false) {
         resolveFrom: __filename,
       },
     ],
-    defaultTargetOptions: {
-      distDir: options.outputDir,
-      publicUrl: "./",
-      shouldScopeHoist: isDev ? false : true,
-    },
+    targets: {
+      storybook: {
+        distDir: options.outputDir,
+        publicUrl: "./",
+        engines: {
```

Comment:
@devongovett are you suggesting we may not need the changes in this file at all anymore?